src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureArea {

```

Comment:
```suggestion
public class Circle extends Figure {
```
it's enough to implement interfaces in Figure